primitives/src/signatures/bls_over_bn254.rs

```diff
@@ -250,6 +260,16 @@ impl PartialEq for VerKey {
     }
 }
 
+// An arbitrary comparison for VerKey. Doesn't mean anything.
+impl PartialOrd for VerKey {
+    fn partial_cmp(&self, other: &Self) -> Option<core::cmp::Ordering> {
+        self.0
+            .into_affine()
```

Comment:
This is a nontrivial computation, no? Is there no way to avoid it?

Reply:
Issue created here: https://github.com/EspressoSystems/jellyfish/issues/457